probers/ldap/probe.go

```diff
@@ -0,0 +1,33 @@
+package ldap
+
+import (
+	"github.com/nmcclain/ldap"
+	"net"
+	"time"
+)
+
+func (p *ldapconfig) probe() error {
+	timeout := time.Duration(p.probefreq) * time.Second
+	timeout = timeout / time.Duration(len(p.hostnames))
+	for _, hostname := range p.hostnames {
+		hostnamePort := hostname + ":636"
+		start := time.Now()
+		conn, err := ldap.DialTLSDialer("tcp", hostnamePort,
+			nil, &net.Dialer{Timeout: timeout})
+		if err != nil {
+			p.healthy = false
+			continue
+		}
+		latency := time.Since(start)
+		p.ldapLatencyDistribution.Add(latency)
```

Comment:
It seems to me that the latency calculation should be done _after_ the Bind is performed. Is there a reason for the way it is now?